x-pack/plugins/apm/public/components/app/transaction_details/distribution/index.tsx

```diff
@@ -122,6 +125,7 @@ export function TransactionDistribution({
           waterfallItemId={waterfallItemId}
           detailTab={detailTab as TransactionTab | undefined}
           waterfall={waterfall}
+          isNotInitiated={isNotInitiated}
           isLoading={isLoading}
           traceSamples={traceSamples}
```

Comment:
A problem I've seen elsewhere is that we break up the return value from `useFetcher` (`data` and `status`) and then pass them down separately. We then come up with new names for these, making the code more complex, and making bugs more likely.

What I suggest: Never pass down `status` or `data` on its own. Always pass down the full `FetcherResult` object. In this case we can replace four props: `waterfall`, `isNotInitiated`, `isLoading` and `traceSamples` with two: `waterfallFetchResult` and `traceSamplesFetchResult`. The receiving component can ofc unpack the result object as needed but if passed further down it should pass the full object.

Would you mind making the change for all components where `waterfall` and `traceSamples` are currently passed?